src/containers/Accounts/AccountHeader/index.tsx

```diff
@@ -317,60 +364,8 @@ const AccountHeader = (props) => {
   )
 }
 
-AccountHeader.propTypes = {
-  language: PropTypes.string.isRequired,
-  onSetCurrencySelected: PropTypes.func.isRequired,
-  currencySelected: PropTypes.string.isRequired,
-  loading: PropTypes.bool.isRequired,
-  accountId: PropTypes.string.isRequired,
-  data: PropTypes.shape({
-    balances: PropTypes.shape({
-      XRP: PropTypes.number,
-    }),
-    paychannels: PropTypes.shape({
-      total_available: PropTypes.string,
-      channels: PropTypes.shape({
-        length: PropTypes.number,
-      }),
-    }),
-    escrows: PropTypes.shape({
-      totalIn: PropTypes.number,
-      totalOut: PropTypes.number,
-    }),
-    signerList: PropTypes.shape({
-      signers: PropTypes.arrayOf(
-        PropTypes.shape({
-          account: PropTypes.string.isRequired,
-          weight: PropTypes.number.isRequired,
-        }),
-      ),
-      quorum: PropTypes.number,
-      maxSigners: PropTypes.number,
-    }),
-    info: PropTypes.shape({
-      reserve: PropTypes.number,
-      sequence: PropTypes.number,
-      ticketCount: PropTypes.number,
-      domain: PropTypes.string,
-      email_hash: PropTypes.string,
-      flags: PropTypes.arrayOf(PropTypes.string),
-      nftMinter: PropTypes.string,
-    }),
-    xAddress: PropTypes.shape({
-      classicAddress: PropTypes.string,
-      tag: PropTypes.oneOfType([PropTypes.number, PropTypes.bool]),
-      test: PropTypes.bool,
-    }),
-    deleted: PropTypes.bool.isRequired,
-  }).isRequired,
-  actions: PropTypes.shape({
-    loadAccountState: PropTypes.func.isRequired,
-  }).isRequired,
-}
-
 export default connect(
-  (state) => ({
-    language: state.app.language,
+  (state: any) => ({
```

Comment:
Why any

Reply:
It is just a mapping and will soon go away when redux is fully removed.  The account and token pages are the last ones using redux for data.  The other redux usages will be removed with the new header.